### DISCUSSION TITLE
MSSQL with FTS

### DISCUSSION THREAD

**Discussion by szekelymatyas** [testcontainers/testcontainers-dotnet#943] [Q&A]
Can someone help me with full text search?
```
var msSqlContainer = new MsSqlBuilder()
        //i need to change the image but keep the configuration 
        // or install fts manually
        .Build();
```
I want to use a different docker file or install fts. Here is what i found so far.
Docker file for fts: 
https://github.com/Microsoft/mssql-docker/blob/1efc4cf9b78fa5fccea682f26067189660af85c8/linux/preview/examples/mssql-agent-fts-ha-tools/Dockerfile
Installing fts:
https://github.com/microsoft/mssql-docker/issues/161#issuecomment-351697672

Any idea?

**Comment by HofmeisterAn** [comment]
Simply build a new image that includes fts and override the MSSQL module image using `WithImage(string)`. In case you need more customizations, you can use the [generic container builder](https://dotnet.testcontainers.org/api/create_docker_container/) too.
